### PR TITLE
Improve print ui

### PR DESCRIPTION
**Pull request by manuelmeister**
before:
Bildschirmfoto 2022-11-20 um 20 02 20

after:
Bildschirmfoto 2022-11-20 um 19 58 58
Bildschirmfoto 2022-11-20 um 19 59 15

### CODE REVIEW

**Review by pmattmann — APPROVED** [review]
I like it :)

**Comment by pmattmann** [comment]
Ich habe mir nur eine Frage gestellt.
Müssen wir den Hintergrund künstlich ausgrauen, damit man die Papiere erkennt.
Aus Neugier habe ich Alternativen ausprobiert.

image

**Comment by pmattmann** [comment]
Ich glaube, mir gefällt diese ganz gut :)

image

**Comment by manuelmeister** [comment]
Ja so etwas könnten wir noch in einem nächsten PR anschauen 👍

Finde es nicht schlecht, noch einmal zu überdenken, ob ein grauer Hintergrund Sinn ergibt oder lieber graue/farbige Seiten.

File: frontend/src/components/print/config/ProgramConfig.vue [hunk 1]
```diff
@@ -37,5 +43,8 @@ export default {
       dayOverview: true,
     }
   },
+  design: {
+    multiple: true,
```
Comment (BacLuc) [inline frontend/src/components/print/config/ProgramConfig.vue:47]: minor:
polymorphism is cool, but i don't know if 5 times a false makes it worth.
I don't have a better idea right  now (except for a "default implementation" with false.

File: frontend/src/components/print/configurator/PagesConfig.vue [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+.e-pages-config:not(.e-pages-config--template) .e-pages-config__title {
+  .v-icon {
+    opacity: 0;
+  }
+
+  &:hover {
+    cursor: move; /* fallback if grab cursor is unsupported */
+    cursor: grab;
+    cursor: -moz-grab;
+    cursor: -webkit-grab;
```
Comment (BacLuc) [inline frontend/src/components/print/configurator/PagesConfig.vue:184]: do we still need these browser specific values?

File: frontend/src/components/print/configurator/PagesConfig.vue [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+  &:active {
+    cursor: move;
+    cursor: -moz-grabbing;
+    cursor: -webkit-grabbing;
+  }
+}
+
+.e-pages-config__delete {
+  min-width: 48px !important;
```
Comment (BacLuc) [inline frontend/src/components/print/configurator/PagesConfig.vue:199]: do we need the important here

File: frontend/src/components/print/configurator/PagesConfig.vue [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+.e-pages-config__delete.invisible {
+  visibility: hidden;
+}
+
+.e-pages-config:hover .e-pages-config__delete {
+  opacity: 1;
+}
+
+.e-pages-config__delete:hover {
+  color: map-get($red, 'base') !important;
```
Comment (BacLuc) [inline frontend/src/components/print/configurator/PagesConfig.vue:213]: do we need the important here

**Review by BacLuc — CHANGES_REQUESTED** [review]
I tested it quickly and had the following findings:
- If you change the order of the pages with drag and drop
  - this is not reflected in the print config for the developer (minor), but may be related to the other two effects
  - nuxt print does not respect the order
  - react print does not respect the order
Am i right that this is just not implemented?
It works on the devel branch

Deleting and adding content works fine

Some other minor things

But it looks good,

**Comment by BacLuc** [comment]
If you want to deploy this branch, you should pull devel into it.
the build and deploy fails because we still reference an unsupported postgres helm chart.
There is a temporary fix on devel.

**Review by BacLuc — APPROVED** [review]

**Pull request merged**